lib/stat_tracker.rb

```diff
@@ -474,5 +462,14 @@ def worst_season(team_id)
     win_percentages.min_by{ |_,  percentage| percentage}&.first
   end
 
+  def biggest_team_blowout(team_id)
```

Comment:
You have this on both the cleanup and its own PR, recommend removing this from cleanup or removing the explicit PR for this